### DISCUSSION TITLE
Fullscreen screen does not work on Mi TV Stick.

### DISCUSSION THREAD

**Discussion by geekytricky** [TeamNewPipe/NewPipe#11039] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

Full-screen screen does not work on Mi TV Stick. please fix it.

**Comment by wb9688** [comment]
Please create a bug report using the template and fill in the relevant details.
